docker/dockerfiles/postgres/Dockerfile

```diff
@@ -0,0 +1,2 @@
+ARG DB_IMAGE_TAG
+FROM postgres:${DB_IMAGE_TAG}
```

Comment:
no, we actually need to download postgres at a particular version (using git clone) and build... it via make